jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/federated/IAMUserAccessKeyFed.java

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright 2019 T Mobile, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ ******************************************************************************/
+/**
+  Copyright (C) 2019 T Mobile Inc - All Rights Reserve
+  Purpose: Rule for checking whether access keys been rotated after a particular duration of days
+  Author : Avinash
+  Modified Date: Jan 25, 2019
+  
+ **/
+package com.tmobile.cloud.awsrules.federated;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTime;
+import org.joda.time.Days;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+import com.tmobile.cloud.awsrules.utils.PacmanUtils;
+import com.tmobile.cloud.constants.PacmanRuleConstants;
+import com.tmobile.pacman.commons.PacmanSdkConstants;
+import com.tmobile.pacman.commons.exception.InvalidInputException;
+import com.tmobile.pacman.commons.rule.Annotation;
+import com.tmobile.pacman.commons.rule.BaseRule;
+import com.tmobile.pacman.commons.rule.PacmanRule;
+import com.tmobile.pacman.commons.rule.RuleResult;
+
+@PacmanRule(key = "check-for-accesskeys-iamuser-federated", desc = "checks for accesskeys for IAM user from current day", severity = PacmanSdkConstants.SEV_HIGH, category = PacmanSdkConstants.GOVERNANCE)
+public class IAMUserAccessKeyFed extends BaseRule {
+	
+	private static final Logger logger = LoggerFactory.getLogger(IAMUserAccessKeyFed.class);
+	
+	/**
+	 * The method will get triggered from Rule Engine with following parameters
+	 * @param ruleParam 
+	 * 
+	 * ************* Following are the Rule Parameters********* <br><br>
+	 * 
+	 * ruleKey : check-for-accesskeys-iamuser-federated-for-180-and-360-days <br><br>
+	 * 
+	 * severity : Enter the value of severity <br><br>
+	 * 
+	 * ruleCategory : Enter the value of category <br><br>
+	 * 
+	 * roleIdentifyingString : Configure it as role/pac_ro <br><br>
```

Comment:
this is no need in this class

Reply:
All the review changes are incorporated.